kroxylicious-operator/src/main/resources/log4j2.yaml

```diff
@@ -13,7 +13,7 @@ Configuration:
       name: STDOUT
       target: SYSTEM_OUT
       PatternLayout:
-        pattern: "%d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c:%L - %m%n"
+        pattern: "%d{yyyy-MM-dd HH:mm:ss} %-5p <%t> %c{2.}:%L - %m%n"
```

Comment:
Is there a reason why the pattern layout here is different to that you've added in `log4j2-test.yaml‎`

Reply:
Not really. 2 chars might be a bit short for prod logs but my assumption being test logs are consumed by developers familiar with the code base and IDEs and thus full package names were much less important there.